src/js/components/presentational/InfiniteTable.jsx

```diff
@@ -91,12 +91,18 @@ class InfiniteTable extends Component {
 
   fontSize (size) {
     switch (size) {
+      case 'Tiny':
```

Comment:
why not using object, with values?

Reply:
@idanstark42 do you want to    fix what Roy said?

Reply:
No. His comment is completely right, but this is a design fix, which I'm not going to do now as a part of this small PR. I would like to do it in a more controlled manner, when I can think about the correct wasy to implement this.